packages/flutter_tools/lib/src/test/event_printer.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert' show JSON;
```

Comment:
I'll use "Chromium Authors" to match the others, but wouldn't "Flutter Authors" be better?